src/UserService.Business/Commands/User/GetUserCommand.cs

```diff
@@ -110,55 +109,27 @@ private PositionInfo GetPosition(Guid userId, List<string> errors)
 
         private List<ProjectInfo> GetProjects(Guid userId, List<string> errors)
         {
-            List<ProjectInfo> result = null;
```

Comment:
Why not update GetUserCommand tests? If they are not there, then you better write

Reply:
Done